js/screens/Onboarding/Onboarding.js

```diff
@@ -50,73 +44,72 @@ let deviceWidth = Dimensions.get('window').width;
 
 const OnboardingPage = () => {
   return (
-    <View>
-      <Onboarding
-        bottomBarHighlight={false}
-        imageContainerStyles={{
-          width: deviceWidth,
-          paddingBottom: 40,
-          paddingTop: 70,
-          justifyContent: 'flex-start',
-        }}
-        bottomBarHeight={100}
-        flatlistProps={{
-          marginHorizontal: 60,
-          width: deviceWidth,
-        }}
-        DotComponent={Circle}
-        SkipButtonComponent={Skip}
-        pages={[
-          {
-            backgroundColor: '#76D2D1',
-            image: (
-              <Image
-                style={{ height: 219, width: 219 }}
-                source={require('../../assets/login/tutorial.png')}
-              />
-            ),
-            title: <Title title="View updates on your child" />,
-            subtitle: <Subtitle subtitle="Description to be added" />,
-            bottomBarHeight: 100,
-          },
-          {
-            backgroundColor: '#76D2D1',
-            image: (
-              <Image
-                style={{ height: 219, width: 219 }}
-                source={require('../../assets/login/fox.png')}
-              />
-            ),
-            title: <Title title="Set up a personalized profile" />,
-            subtitle: <Subtitle subtitle="Description to be added" />,
-          },
-          {
-            backgroundColor: '#76D2D1',
-            image: (
-              <Image
-                style={{ height: 219, width: 219 }}
-                source={require('../../assets/login/provider.png')}
-              />
-            ),
+    <Onboarding
+      bottomBarHighlight={false}
+      showNext={false}
+      showDone={false}
+      imageContainerStyles={{
+        width: deviceWidth,
+        paddingBottom: 40,
+        paddingTop: 70,
+        justifyContent: 'flex-start',
+      }}
+      bottomBarHeight={100}
+      flatlistProps={{
+        width: deviceWidth,
+      }}
+      DotComponent={Circle}
+      SkipButtonComponent={Skip}
+      pages={[
+        {
+          backgroundColor: '#76D2D1',
+          image: (
+            <Image
+              style={{ height: 219, width: 219 }}
+              source={require('../../assets/login/tutorial.png')}
+            />
+          ),
+          title: <Title title="View updates on your child" />,
+          subtitle: <Subtitle subtitle="Description to be added" />,
+          bottomBarHeight: 100,
+        },
```

Comment:
Looks like there are a lot of similarities between your pages markup here. Please make a component and do the markup once only.

Reply:
The background colour on onBoarding page will be assigned after the GlobalStyles colours are defined. (Will be with Cody's PR)